doc/en/lazy_load.md

```diff
@@ -25,17 +30,17 @@ spec:
   image:
     pullPolicy: Always
     repository: docker.io/slimeio/slime-lazyload
-    tag: v0.2.0-alpha
+    tag: {{your_lazyload_tag}}
   module:
     - name: lazyload
       fence:
         enable: true
         wormholePort: 
           - "9080" # replace to your application service ports
```

Comment:
"and extend the list in case of multi ports"

Reply:
Done.